### PR TITLE
Accessibility improvements

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Fixed a few accessibility issues showing up on the browser devtools

Footer.js has `socialLinks` array with four links using icons and lacking alt text. I tried passing in `alt` text to these links, as well as directly to the icon, but wasn't able to get it to work, so that's the only one left.

## Related Issue
ethereum.org:
image

this PR localhost:
image

...btw @samajammin @ryancreatescopy , SEO and Best Practices rank 100%

### CODE REVIEW

File: src/components/Nav/Mobile.js [hunk 1]
```diff
@@ -244,15 +244,14 @@ const MobileNavMenu = ({
                   <SectionItems>
                     {section.items.map((item, idx) => {
```
Comment (Kristiyan96) [inline src/components/Nav/Mobile.js:245]: No need for the return statement. 
```
section.items.map((item, idx) => (
  <SectionItem key={idx} onClick={() => toggleMenu()}>
     ...
  </SectionItem>
)
```

File: src/components/Nav/Mobile.js [hunk 2]
```diff
@@ -244,15 +244,14 @@ const MobileNavMenu = ({
...
                       return (
-                        <StyledNavLink
-                          to={item.to}
-                          isPartiallyActive={item.isPartiallyActive}
-                          key={idx}
-                        >
-                          <SectionItem onClick={() => toggleMenu()}>
+                        <SectionItem key={idx} onClick={() => toggleMenu()}>
```
Comment (Kristiyan96) [inline src/components/Nav/Mobile.js:247]: onClick={toggleMenu}

**Review by Kristiyan96 — CHANGES_REQUESTED** [review]
Some small improvements

File: src/components/Nav/Mobile.js (hunk 1 above)
Comment (wackerow) [reply src/components/Nav/Mobile.js:245]: Agreed, these are scattered throughout the codebase and could be cleaned up in batch in a separate issue. I'll push the syntax fix here though while we're looking at it.

**Review by samajammin — APPROVED** [review]

**Pull request merged**